examples/custom_model.py

```diff
@@ -0,0 +1,117 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
```

Comment:
On one of these slow days, we could search and replace the 2021 copyright statement and update it to 2022.

Reply:
Actually I was told by the lawyers to keep the 2021 copyright year because that's when we received permission to first release. 

It surprised me too